geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ExportStackTraceCommand.java

```diff
@@ -50,6 +53,10 @@
   public static final String STACK_TRACE_FOR_MEMBER = "*** Stack-trace for member ";
   private final GetStackTracesFunction getStackTracesFunction = new GetStackTracesFunction();
 
+  private static final DateTimeFormatter formatter =
+      DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm:ss.SSS", Locale.getDefault())
+          .withZone(ZoneId.systemDefault());
```

Comment:
The `Locale.getDefault()` parameter and the further customisation through `.withZone(ZoneId.systemDefault())` are not needed, you can achieve the same by just executing`DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm:ss.SSS")`.

Reply:
It is possible to go without `Locale.getDefault()`, but we need to add `.withZone(ZoneId.systemDefault())`. To format an `Instant` a time-zone is required. Without a time-zone, the formatter does not know how to convert the instant to human date-time fields, and therefore throws an exception.